### DISCUSSION TITLE
⏳ ⛔ Allow that a whole section/container can be saved as a Global Template.

### DISCUSSION THREAD

**Discussion by JavierRzRd** [elementor/elementor#21166] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Is an uphill process when changing a section of a website and you have to save a Section as a template then do a Delete-Insert Template process on every page that the section exists. For example: Call to Actions or standard information.

### Describe the solution you'd like

Allow us to save a whole section as a template. If we make a change to the whole structure (texts, colors, new elements added...) the whole website will have the changes applied automatically on pages using that Section Template.

**Comment by gervy28** [comment]
This would be such a massive feature that would make developing reusable blocks so much easier. 

To add to the **"Describe Alternatives You've Considered"** 
The current work around is to create a template, then to insert that template into a shortcode block. This sort of works, but it also produces extra containers and can be challenging to get the layout right.

**Reply by eyad-alkhalidy** [reply]
I apply the same workaround, but instead of using the "Shortcode" widget, I use the "**Template**" widget which allows me to **select** one of the saved templates; no shortcode copy/paste is needed this way, and you have a **direct link to edit** that selected template. However, it has the same exact results, where this template will be inserted into a container/section. To achieve the desired result, I always make this parent container/section full width and remove its default padding.

**Reply by nolderoos** [reply]
This is the workflow I see most developers using @eyad-alkhalidy, it works, I think its just a matter of using one default for global for Elementor. They've got to choose between Global Elements and Templates

**Comment by tnchuntic** [comment]
Agree with this request, as of now I can't save container as Global widget. 

Only the elements inside the container can be save as Global which is a bit weird because of Container are use to layout the contents inside.

**Comment by Masti81** [comment]
Truely inexplicable how this is not a feature. I have a site with over 50 pages and i simply want to add CTAs to a lot of the pages. There will likely need tobe tweaks to the CTAs in future. It is highly ineffiecent to have to manually change the CTA on dozens of pages. Would love this added as a feature. Beaver Builder had this yonks ago.

**Reply by loucasl** [reply]
Just save the container or section as a template and reuse it via the template widget or shortcode wherever you want

**Reply by romanoooooooooo** [reply]
If we update the template, the corresponding section reused on the site doesn't get updated. However, with the shortcode, it's possible. Thanks for the idea!!

**Comment by kaelsense** [comment]
It should be possible to save a section as a Global Widget. There should not be any need to do any kind of tricks to achieve this.

**Comment by medieskolenlyngby** [comment]
Yes please

**Comment by ginnyweaver** [comment]
Agree, this is madness, why is this not a feature?? I just used one of my developer licenses to upgrade a site to Pro, thinking I could save a container as a Global Widget, but apparently not. This is just really, really dumb.

**Comment by PaulinaJalou** [comment]
..... still waiting for this feature.... please Elementor, come on... let us save a whole container as global widget!!!

**Comment by klitsios** [comment]
please Elementor, give us the option to save a whole container as global widget!

**Comment by Joker2099** [comment]
Just like Loucasl said, save the container as a template and use that template's Shortcode on all pages as needed. It's the same exact thing.

**Comment by lcgitrdone** [comment]
Same here.  Not intuitive.  Thank you!

**Comment by nmbrthirteen** [comment]
+1 This feature, along with #22203 will be the most significant jump and improvement. Thanks for this idea. Hope they will implement this.

**Comment by klitsios** [comment]
Got tired of waiting for a fix, so I switched all my projects to Bricks and canceled my Elementor Pro subscription.

**Comment by siegel-websites** [comment]
such an important feature! please make it easier to use.

**Comment by stencells** [comment]
+1

**Comment by rendermouse** [comment]
It is frustrating that you cannot save a container as global. You cannot add a widget to a page without a container. So you have to add a container, then drop a template widget into it to make it act as a global. Your global component's container can therefore never be the outermost element, like a section element would be, it will always require a parent container.

**Comment by amjulash** [comment]
+1

**Comment by ellemaker** [comment]
why does the developer doesn't implement this at the very first version? lol
